app/src/main/java/com/ybsystem/tweethub/adapters/array/DrawerArrayAdapter.java

```diff
@@ -0,0 +1,173 @@
+package com.ybsystem.tweethub.adapters.array;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.ybsystem.tweethub.R;
+import com.ybsystem.tweethub.activities.SearchActivity;
+import com.ybsystem.tweethub.activities.TimelineActivity;
+import com.ybsystem.tweethub.activities.UserListActivity;
+import com.ybsystem.tweethub.activities.preference.SettingActivity;
+import com.ybsystem.tweethub.application.TweetHubApp;
+import com.ybsystem.tweethub.models.enums.ColumnType;
+import com.ybsystem.tweethub.storages.PrefAppearance;
+import com.ybsystem.tweethub.utils.CalcUtils;
+import com.ybsystem.tweethub.utils.DialogUtils;
+
+import static com.ybsystem.tweethub.models.enums.ColumnType.*;
+
+public class DrawerArrayAdapter extends ArrayAdapter<Integer> {
+
+    public DrawerArrayAdapter(Context context) {
+        super(context, android.R.layout.simple_list_item_1);
+    }
+
+    @Override
+    public View getView(int position, View convertView, ViewGroup parent) {
+        // Inflate
+        View cv = convertView;
+        LayoutInflater inflater = LayoutInflater.from(getContext());
+
+        if (position == 0 || position == 13) {
+            cv = inflater.inflate(R.layout.list_item_divider, parent, false);
+            cv.findViewById(R.id.view_divider).setVisibility(View.GONE);
+            cv.setEnabled(false);
+            cv.setOnClickListener(null);
+
+        } else if (position == 6 || position == 10) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "cv". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS1854)